src/main/java/com/jvm_bloggers/utils/DateTimeUtilities.java

```diff
@@ -14,6 +14,10 @@
 
 public class DateTimeUtilities {
 
+    private DateTimeUtilities() {
+        throw new IllegalStateException("Utility class");
+    }
+
```

Comment:
We have a Lombok dependency in the project already. We can use [`@UtilityClass`](https://projectlombok.org/features/experimental/UtilityClass) annotation instead.
```suggestion
@UtilityClass
public class DateTimeUtilities {
```

Reply:
Thank you. Done :)